src/ReactiveUI/Platforms/android/AndroidObservableForWidgets.cs

```diff
@@ -142,10 +142,14 @@ private static DispatchItem CreateTimePickerMinuteFromWidget()
                 {
                     var view = (TView)x;
 
-                    return Observable.FromEvent<EventHandler<TEventArgs>, TEventArgs>(
```

Comment:
This was the cause of the error. The event handlers weren't being set properly.

Reply:
Essentially, there was a mismatch between the EventHandler and Rx would cause an exception.

Now we are explicitly specifying the event handler method which is more efficient since it avoids reflection anyway.